src/pages/index.js

```diff
@@ -204,7 +204,7 @@ profileEditButton.addEventListener("click", () => {
 /*                        Form Popup : Adding New Card                        */
 /* -------------------------------------------------------------------------- */
 
-/* function handleNewCardServerRenderSubmit() {
+function handleNewCardServerRenderSubmit() {
   submitButtonAddNewCard.textContent = "Saving...";
   api
     .addNewCard({
```

Comment:
```js

      name: newCardNameInput.value,
      link: newCardImageUrlInput.value,
```

* Only method `_getInputValues` collects all the data of the inputs and passes it to the call of `submitForm`. So you need to use the collected data rather than the inputs.